source/_views/partials/tags.html

```diff
@@ -0,0 +1,7 @@
+<!-- array of post authors: this is where new authors are added.  Add the author into the set authors array as "first-last"-->
+{% set authors = ["nikki-stevens", "jason-savino", "mike-klass"] %} 
+<!-- conditional statement to skip author tags from appearing in posts and tags page -->
+{% if tag in authors %}
```

Comment:
does twig support NOT in authors so you don't have to do an if and then also an else?